src/test/java/core/basesyntax/service/CsvUniqueFruitsAdderServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.impl.CsvFileReaderServiceImpl;
+import core.basesyntax.service.impl.CsvFileUniqueFruitsServiceAdderImpl;
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class CsvUniqueFruitsAdderServiceTest {
+    private static final String VALID_FILE_PATH = "src/main/resources/input.csv";
+    private static final String NON_VALID_STRUCTURE_FILE_PATH =
+            "src/main/resources/non-validStructureInTransactions.csv";
+    private static final String NON_VALID_STRATEGY_FILE_PATH =
+            "src/main/resources/nonValidStrategyFile.csv";
+    private static final String NON_VALID_STRUCTURE_IN_TRANSACTION_FILE_PATH =
+            "src/main/resources/non-validStructureInTransactions.csv";
+    private static final String NON_VALID_QUANTITY_FILE_PATH =
+            "src/main/resources/non-validQuantity.csv";
+    private Storage storage = new Storage();
+    private UniqueFruitsAdderService uniqueFruitsAdderService
+            = new CsvFileUniqueFruitsServiceAdderImpl(storage);
+    private FileReaderService fileReaderService = new CsvFileReaderServiceImpl();
+
+    @Test
+    void validSolutionTest() {
```

Comment:
Please unify all test method names to follow convention
https://github.com/mate-academy/jv-fruit-shop-tests/blob/main/checklist.md#remember-about-the-naming-of-test-methods